packages/azure/changelog.yml

```
@@ -1,3 +1,8 @@
+- version: "1.24.0"
+  changes:
+    - description: Map `service.id` and `device.id` ECS fields to the `*.properties.service_principal_id` and `*.properties.c_device_id` fields, respectively.
```

I would interpret this to mean the opposite of what is happening here. This suggests `service.id` → `*.properties.service_principal_id`, but what we have is `*.properties.service_principal_id` → `service.id`.
```suggestion
    - description: Map `*.properties.service_principal_id` and `*.properties.c_device_id` to `service.id` and `device.id`, respectively.
```

---

Agreed. The existing commit message and changelog description meant the opposite.